tests/cmsis_rtos_v1/src/mutex.c

```diff
@@ -101,6 +101,9 @@ void tThread_entry_lock_timeout(void const *arg)
 	status = osMutexWait((osMutexId)arg, TIMEOUT - 100);
 	zassert_true(status == osErrorTimeoutResource, NULL);
 
+	status = osMutexRelease((osMutexId)arg);
```

Comment:
can we please add a comment mentioning, what the code snippet is validating ?
something like "Mutex Ownership check - try to release a mutex, acquired by another thread"

Reply:
Addressed.